### ISSUE TITLE
 youtube-dl doesn't download 1080p video

### ISSUE THREAD

**Issue by rulet** [rg3/youtube-dl#2165]
Will it be fixed?

**Comment by houdinialex** [comment]
youtube-dl -f 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/bestvideo+bestaudio' --merge-output-format mp4 [LINK]

**Comment by HVEC4** [comment]
Hi guys, this is super late but I've got youtube-dl and ffmpeg for the first time today, and I'm also seeing this, where I can only download merged up to 720p, and separately (and merge later with ffmpeg) at 1080p. This is fine, and ffmpeg automatically merges them if I do it as -f 137+140, but every time I try, the video is frozen after 5 seconds and the audio continues. I tried downloading them separately and I'll just layer them myself in iMovie or something (-f 140 [link], and then afterward -f 137 [link]), but sure enough, the audio file is fine but the video is 5 seconds long. What am I doing wrong? I've done this at least 20 times today but cannot get the video longer than 5 seconds long. It is a video that is over 12 minutes long that I am trying to get here. Someone help?
